fv3core/stencils/updatedzd.py

```diff
@@ -77,10 +78,19 @@ def zh_damp_stencil(
     fy2: FloatField,
     rarea: FloatFieldIJ,
     zh: FloatField,
+    zs: FloatFieldIJ,
+    ws: FloatFieldIJ,
+    dt: float,
 ):
```

Comment:
Can you add a docstring for this? It's another one that's particularly unclear to me what it's doing and what some of the inputs are.